src/Apps/Order/Routes/Status/index.tsx

```diff
@@ -341,6 +341,7 @@ export const StatusFragmentContainer = createFragmentContainer(
         lineItems {
           edges {
             node {
+              ...ItemReview_lineItem
```

Comment:
looks like this fragment is not actually used and can be deleted anyway. Same goes for the `artwork` selection below.

Reply:
just checked whether they were being used in this file (they were not) and then check whether any of the places we passed 'order' to were not fragment containers (they were all fragment containers).